miden-crypto/src/dsa/rpo_falcon512/mod.rs

```diff
@@ -68,21 +122,33 @@ type ShortLatticeBasis = [Polynomial<i16>; 4];
 pub struct Nonce([u8; SIG_NONCE_LEN]);
 
 impl Nonce {
-    /// Returns a new [Nonce] instantiated from the provided bytes.
-    pub fn new(bytes: [u8; SIG_NONCE_LEN]) -> Self {
-        Self(bytes)
+    /// Returns a new [Nonce].
+    pub fn preversioned() -> Self {
```

Comment:
nit: I would maybe expand the comment to explain that this returned "preversioned" nonce (and what that means).